apps/frontend/src/components/Sidebar.tsx

```diff
@@ -147,22 +97,19 @@ const Sidebar: React.FC = () => {
   })
 
   return (
-    <div className="mx-4 my-auto flex select-none flex-col items-center justify-center rounded-[30px] border border-border bg-background px-2 py-8">
-      <motion.div
-        className="flex flex-col gap-2"
-        onMouseMove={(e) => mouseY.set(e.pageY)}
-        onMouseLeave={() => {
-          mouseY.set(Infinity); 
-          setIsHovered(false);  
-        }}
-        onMouseEnter={() => setIsHovered(true)}  
-      >
+    <div
+      className={classNames(
+        "mx-4 my-auto flex select-none flex-col rounded-[30px] border border-border px-2 transition-all",
+        holdIsExpanded ? "py-8" : "pt-8 pb-2"
```

Comment:
Instead of this you can use   const expandedClassName = isExpanded ? "py-8" : "pt-8 pb-2" and use expandedClassName whereever needed

Reply:
Or just use  isExpanded ? "py-8" : "pt-8 pb-2"